src/utils/helpers/list/buildQueryFilters.ts

```diff
@@ -10,14 +10,14 @@ interface NamedDateMap {
   [key: string]: string[]
 }
 
-export default function buildQueryFilters(filters: BasicStringObject) {
+export default function buildQueryFilters(filters: BasicStringObject, userId: number) {
   const graphQLfilter = Object.entries(filters).reduce((filterObj, [key, value]) => {
     if (!mapQueryToFilterField[key]) return filterObj
     return { ...filterObj, ...mapQueryToFilterField[key](value) }
   }, {})
   // If no filters, return a dummy filter to prevent GraphQL empty object error
-  if (Object.keys(graphQLfilter).length === 0) return { templateCode: { isNull: false } }
-  return graphQLfilter
+  if (Object.keys(graphQLfilter).length === 0) return { templateCode: { isNull: true } }
```

Comment:
This is just a error-protection because graphQL can't accept empty filter objects. It was meant to return everything if no filter object was supplied, but with `isNull: true` it'll return nothing. Since the next line adds a reviewerId to the filter object, we probably don't need this line (maybe comment it out for now?)